src/main/java/com/manardenza/Main.java

```diff
@@ -1,5 +1,6 @@
 package com.manardenza;
 
+import com.manardenza.controller.HotelController;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.manardenza.controller.HotelController'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)